app/brave_generated_resources.grd

```diff
@@ -230,6 +230,11 @@
           Restart browser to enable Google Widevine
         </message>
       </if>
+      <if expr="is_android">
+        <message name="IDS_WIDEVINE_PERMISSION_REQUEST_TEXT_FRAGMENT_ENABLE_SYSTEM" desc="Text fragment for Widevine permission request. 'Widevine' is the name of a plugin and should not be translated.">
+          <ph name="URL">$1<ex>https://www.youtube.com</ex></ph> wants to play protected Google Widevine content, which requires you to allow Android system support for Widevine. By allowing, you'll agree to Google's terms of use.
```

Comment:
Why are we specifically referencing youtube here? There are any number of sites that require widevine

Reply:
The part between `<ex>` is an example, which will not be displayed and in the actual UI it will display the url of the current webpage.